lib/ddio.h

```diff
@@ -320,14 +321,25 @@ void ddio_MouseSetVCoords(int width, int height);
 //	File Operations
 //	---------------------------------------------------------------------------
 
-//	creates or destroys a directory or folder on disk
-//	This pathname is *RELATIVE* not fully qualified
-bool ddio_CreateDir(const char *path);
-bool ddio_RemoveDir(const char *path);
+//	creates a directory on disk
+inline bool ddio_CreateDir(std::filesystem::path const& path) {
+  std::error_code ec{};
+  std::filesystem::create_directory(path, ec);
+  return !ec;
+}
+
+inline bool ddio_RemoveDir(std::filesystem::path const &path) {
+  std::error_code ec{};
+  std::filesystem::remove(path, ec);
+  return !ec;
+}
 
 // deletes a file.  Returns 1 if successful, 0 on failure
-//	This pathname is *RELATIVE* not fully qualified
-int ddio_DeleteFile(char *name);
+inline int ddio_DeleteFile(std::filesystem::path const &path) { 
+  std::error_code ec{};
+  std::filesystem::remove(path, ec);
+  return ec ? 0 : 1;
```

Comment:
Well, I think we can change function signature here to `bool ddio_DeleteFile(std::filesystem::path const &path)`

Reply:
haha yeah, my initial goal a couple days ago was to do this very piecemeal, but have since become more aggressive with the changes